[BE] Add bio for user

nspc97 opened the pull request:
[BE] Add bio for user
https://trello.com/c/Ot8ZwNv4/81-be-add-bio-for-user

- Added user biography
- User is now unable to change username

Commit:
[BE] Add bio for user
https://trello.com/c/Ot8ZwNv4/81-be-add-bio-for-user

- Added user biography
- User is now unable to change username

MihaiDruta commented on src/DTO/UserDTO.php:
```diff
@@ -146,6 +146,17 @@ class UserDTO
+     * User biography
+     * @var string
+     * @Serializer\Type("string")
+     * @Assert\NotBlank(groups={"UserEdit"})
```
No need of `@Assert\NotBlank(groups={"UserEdit"})`. Imho, biography should be optional

Commit:
[BE] Add bio for user
https://trello.com/c/Ot8ZwNv4/81-be-add-bio-for-user

- Removed not blank assert

Commit:
[BE] Add bio for user
https://trello.com/c/Ot8ZwNv4/81-be-add-bio-for-user

- User is now unable to change email
